samples/immutable-messages/using-classes/Core_7/Receiver/NonPublicPropertiesResolver.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Reflection;
+
+public class NonPublicPropertiesResolver : DefaultContractResolver
```

Comment:
Why do you add this? Deserializing to private setters worked for me out of the box.

Reply:
Because this one is one of those strange case where I cannot reliably get it working. If it worked for you without the custom resolver then I’ll remove it.

Based on this do you think we need both samples?